src/DB_Command.php

```diff
@@ -495,24 +495,40 @@ public function cli( $_, $assoc_args ) {
 	 *     +---------+-----------------------+
 	 */
 	public function query( $args, $assoc_args ) {
-
 		$command = sprintf( '/usr/bin/env mysql%s --no-auto-rehash', $this->get_defaults_flag_string( $assoc_args ) );
 		WP_CLI::debug( "Running shell command: {$command}", 'db' );
-
 		$assoc_args['database'] = DB_NAME;
 
-		// The query might come from STDIN.
```

Comment:
Comments unrelated to the specific section that is being changed should not be removed